### ISSUE TITLE
12 button mode

### ISSUE THREAD

**Issue by Dejal** [samuelclay/turntouch-mac#9]
Should add support for 12 button mode in the Mac app.

**Comment by samuelclay** [comment]
Agreed, I've long wanted to do this. not sure how easy it would be for you to do but I basically worked on this during Summer 2017 on iOS and you can follow the changes pretty easily.
